### PR TITLE
handle review comments for PR 2784

### PR DESCRIPTION
**Pull request by nishantmonu51**
follow up PR for review comment - 
https://github.com/druid-io/druid/pull/2784#discussion_r59062021

### CODE REVIEW

**Comment by xvrl** [comment]
👍

File: docs/content/configuration/index.md [hunk 1]
```diff
@@ -296,7 +296,6 @@ In current Druid, multiple data segments may be announced under the same Znode.
 |--------|-----------|-------|
 |`druid.announcer.segmentsPerNode`|Each Znode contains info for up to this many segments.|50|
 |`druid.announcer.maxBytesPerNode`|Max byte size for Znode.|524288|
-|`druid.announcer.skipDimensions`|Skip Dimension list from segment announcements. NOTE: Enabling this will also remove the dimensions list from coordinator and broker endpoints.|false|
-|`druid.announcer.skipMetrics`|Skip Metrics list from segment announcements. NOTE: Enabling this will also remove the metrics list from coordinator and broker endpoints.|false|
+|`druid.announcer.skipDimensionsAndMetrics`|Skip Dimensions and Metrics list from segment announcements. NOTE: Enabling this will also remove the dimensions and metrics list from coordinator and broker endpoints.|false|
```
Comment (fjy) [inline docs/content/configuration/index.md]: do we still have this info available via segment metadata query?

Comment (nishantmonu51) [inline docs/content/configuration/index.md]: yeah.

Comment (xvrl) [inline docs/content/configuration/index.md]: @fjy this PR is not changing anything, #2784 is the one that made the changes, and you +1 that one.

**Comment by fjy** [comment]
👍

**Pull request merged**